client/my-sites/plans-features-main/index.jsx

```diff
@@ -315,29 +335,67 @@ class PlansFeaturesMain extends Component {
 		);
 	}
 
+	getIntervalTypeToggle() {
+		const {
+			translate,
+			intervalType,
+			site,
+			basePlansPath,
+		} = this.props;
+		const segmentClasses = classNames(
+			'plan-features__interval-type',
+			'price-toggle'
+		);
+
+		let plansUrl = '/plans';
+		if ( basePlansPath ) {
+			plansUrl = basePlansPath;
+		}
+
+		return (
+			<SegmentedControl compact className={ segmentClasses } primary={ true }>
+				<SegmentedControlItem
+					selected={ intervalType === 'monthly' }
+					path={ plansLink( plansUrl, site, 'monthly' ) }
+				>
+					{ translate( 'Monthly billing' ) }
+				</SegmentedControlItem>
+
+				<SegmentedControlItem
+					selected={ intervalType === 'yearly' }
+					path={ plansLink( plansUrl, site, 'yearly' ) }
+				>
+					{ translate( 'Yearly billing' ) }
+				</SegmentedControlItem>
+			</SegmentedControl>
+		);
+	}
+
 	render() {
 		const {
 			site,
-			showFAQ,
-			displayJetpackPlans
+			displayJetpackPlans,
+			isInSignup,
+			isInSignupTest
 		} = this.props;
 
 		const renderFAQ = () =>
 			displayJetpackPlans
 				? this.getJetpackFAQ()
 				: this.getFAQ( site );
+		let faqs = renderFAQ();
+
+		if ( this.isInSignupTest() || ( ! displayJetpackPlans && isInSignup ) ) {
+			faqs = null;
+		}
```

Comment:
Can you update this logic so `faqs` is initialized with `null`? It would be better to avoid unnecessary function callings.